### ISSUE TITLE
chewie does not respect previous UI orientation

### ISSUE THREAD

**Issue by fbatschi** [brianegan/chewie#61]
When chewie enters Fullscreen mode, it set the preferred Orientation to landscape mode.
when you exits fullscree, chewie sets the preferred orientation ot portrait as well as landscape.

It is fine to have landscape mode for the video, but if you had your app set to portrait mode only before, after leaving fullscreen video mode, the app will be available in landscape mode again, which might not be intended.

chewie should respect the previous orientation mode.

**Comment by fbatschi** [comment]
We have looked into this further and _pushFullScreenWidget() in lib/src/chewie_player.dart is setting all available DeviceOrientations after full screen mode. This erases the settings when you have locked the app to Portrait mode for instance.

```
SystemChrome.setPreferredOrientations([
      DeviceOrientation.portraitUp,
      DeviceOrientation.portraitDown,
      DeviceOrientation.landscapeLeft,
      DeviceOrientation.landscapeRight,
    ]);
```
Unfortunately there is no `SystemChrome.getPreferredOrientations()` so that chewie will not be able to remember the previous preferred device orientations. 

The only way to fix this right now is that we introduced a new parameter to an extended Chewie class where we set the preferredOrientations.

If you want to, i can send a PR.

**Comment by brianegan** [comment]
Yes please!  A PR for this setting would be great :)

**Comment by cbenhagen** [comment]
I added an option to set the allowed device orientations after exiting full screen in 0da55825608afc1b262fc5a2d3f0ff79fc7a149f. This has been released in chewie 0.9.2.

You can set it like this:
```dart
final chewieController = ChewieController(
  videoPlayerController: videoPlayerController,
  aspectRatio: 3 / 2,
  autoPlay: true,
  looping: true,
  deviceOrientationsAfterFullScreen: [DeviceOrientation.portraitUp]
);
```

**Comment by thivankasarathchandra** [comment]
Guys I'm very new to the github and just letting know, right now full screen button let us turn the orientation back. but when we click device back button orientation will lock. We can rest the orientation by 

```
import 'package:flutter/services.dart';
@override
  void dispose() {
    SystemChrome.setPreferredOrientations([
    DeviceOrientation.portraitUp,
  ]);
    _videoPlayerController1.dispose();
    _chewieController.dispose();
    super.dispose();
  }
```

Sorry I just wanted to let you know and didn't know where to put this.
